Tests/UITests/UITests.xcodeproj/xcshareddata/xcschemes/TestApp.xcscheme

```diff
@@ -37,7 +37,7 @@
       </BuildActionEntries>
    </BuildAction>
    <TestAction
-      buildConfiguration = "Debug"
+      buildConfiguration = "Release"
```

Comment:
Not sure if this is intentional here to use release builds for testing.

Reply:
Ah, what a misclick. Intention was to change it to "Testing" 😅